v2/client_test.go

```diff
@@ -53,9 +64,10 @@ type nopCloser struct {
 func (nopCloser) Close() error { return nil }
 
 type httpChecks struct {
-	URL    string
-	body   string
-	params map[string]string
+	URL     string
+	body    string
+	params  map[string]string
+	headers map[string]string
```

Comment:
never mind - I thought this was defining something different, not a test infrastructure thingy.